hax/hax/util.py

```diff
@@ -449,6 +449,17 @@ def get_leader_session_no_wait(self) -> str:
             raise HAConsistencyException(
                 'Could not get the leader from Consul')
 
+    def is_leader_value_present_for_session(self) -> bool:
+        leader = self.kv.kv_get('leader')
+        if leader is None:
+            raise HAConsistencyException('No leader key exists yet')
+
+        node: bytes = leader['Value']
+        if node is None:
+            return False
+
```

Comment:
[*style*] extra blank line not required.

Reply:
Done